pkg/workload/workload.go

```diff
@@ -675,9 +661,5 @@ func References(wls []*Info) []klog.ObjectRef {
 	if len(wls) == 0 {
 		return nil
 	}
-	keys := make([]klog.ObjectRef, len(wls))
-	for i, wl := range wls {
-		keys[i] = klog.KObj(wl.Obj)
-	}
-	return keys
+	return utilslices.Map(wls, func(pi **Info) klog.ObjectRef { return klog.KObj((*pi).Obj) })
```

Comment:
I think this place is now harder to read, so I don't see any gain. I would suggest to revert.